jetty-core/jetty-deploy/src/main/java/org/eclipse/jetty/deploy/StandardContextHandlerFactory.java

```diff
@@ -292,10 +294,17 @@ private boolean applyEnvironmentXml(Server server, Object context, Environment e
         return true;
     }
 
-    private ClassLoader getClassLoader(Object context, Environment environment)
+    private ClassLoader getClassLoader(Object context, Attributes attributes, Environment environment) throws IOException
```

Comment:
The 'context' parameter seems to not necessarily be a context, in which case it isn't well named.  This needs some javadoc to explain what it can be and what that means.